seminars/2024-02-07.qmd

```diff
@@ -1,6 +1,10 @@
 ---
-title: "Global diabetes burden. Prevalence, incidence and mortality from type 2 diabetes, type 1 diabetes, and gestational diabetes"
+title: "Carga global de diabetes tipo 2. Prevalencia, incidencia y mortalidad"
+subtitle:"Global  burden of type 2 diabetes. Prevalence, incidence and mortality"
+description: "add here a short description of the seminar"

```

Comment:
```suggestion
# description: "add here a short description of the seminar"
```

This will show up with this description, so I am just commenting it out for now.